site/content/api/Tenant-API.md

```diff
@@ -0,0 +1,264 @@
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+**Response Message Format**
+
+A response to a *get tenant information* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message includes payload as defined in the [Tenant Format]({{< relref "#tenant-format" >}}) section below.
+
+The response message's *status* property may contain the following codes:
+
+| Code | Description |
+| :--- | :---------- |
+| *200* | OK, the payload contains the tenant information for the requested tenant. |
+| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+## Update Tenant
+
+Clients use this command to *update* information about an already existing tenant. All of the information that has been previously provided for the tenant gets *replaced* with the information contained in the request message.
+
+This operation is *optional*, implementors of this API may provide other means for updating tenant information, e.g. a RESTful API or a configuration file.
+
+**Message Flow**
+
+The following sequence diagram illustrates the flow of messages involved in a *Client* updating tenant information.
+
+![Update Tenant Information message flow](../tenant_UpdateTenantSuccess.png)
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on an *update tenant* message in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name        | Mandatory | Location                 | Type     | Description |
+| :---------- | :-------: | :----------------------- | :------- | :---------- |
+| *subject*   | yes       | *properties*             | *string* | MUST be set to `update`. |
+
+The request message MUST include payload as defined in the [Tenant Format]({{< relref "#tenant-format" >}}) section below.
+ 
+**Response Message Format**
+
+A response to an *update tenant* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message's *status* property may contain the following codes:
+
+| Code  | Description |
+| :---- | :---------- |
+| *204* | No Content, the tenant has been updated successfully. |
+| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+## Remove Tenant
+
+Clients use this command to *remove* all information about a tenant from Hono. Once a tenant is removed, clients can no longer use Hono to interact with any device of the tenant nor can they consume any more data produced by the devices of the tenant.
```

Comment:
oh, here is some information on the impact. :+1: 
we should add that no device belonging to the tenant can upload data anymore as well ...

Reply:
I added some more lines to make the consequences more clear.